pkg/prometheus/watcher_test.go

```diff
@@ -0,0 +1,117 @@
+package prometheus
+
+import (
+	"flag"
+	"reflect"
+	"testing"
+
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+)
+
+func init() {
```

Comment:
not a biggie: you can use TestMain function to initialize stuff

Reply:
Done